rootfs/rootfs/etc/rc.d/vmtoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/local/bin/vmtoolsd --background /var/run/vmtoolsd.pid
```

Comment:
I'm positive that `vmtoolsd` would quit if can't talk to the vmx backdoor, but let me test it on VirtualBox to be 100% sure.